scripts/run-prod.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ev
+
+echo "${TRAVIS_BRANCH}"
+echo "${TRAVIS_PULL_REQUEST}"
+echo "${TRAVIS_PULL_REQUEST_BRANCH}"
+echo "${TRAVIS_PULL_REQUEST_SHA}"
+
+if [ ${#RUN_PROD} == "true" ]; then
+  ng build --prod
```

Comment:
While we're at it, run "npm shrinkwrap" as well to ensure that that doesn't break too.